src/pages/gen2/build-a-backend/functions/extend-with-cdk/index.mdx

```diff
@@ -0,0 +1,17 @@
+export const meta = {
+  title: 'Extend with CDK',
+  description:
+    'Learn how to extend the underlying Function resources'
+};
+
+export function getStaticProps() {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+## Adding IAM Policies
```

Comment:
Does this title not better belong in the access rules section?

Reply:
this is for IAM policies beyond the access prop, but I plan to call it out in the access rules section. mostly a note of circular dependencies